### ISSUE TITLE
Trial Jax Template API and retire custom loader

### ISSUE THREAD

**Issue by Afollet** [PDXFinder/pdxfinder#378]
As a Developer, I would like to maintain the least amount of code and retire deprecated code.

Jax has developed an API for the PDX Finder template. The api should be scrapped and then loaded in the PDX Finder to test for inconsistency and mistakes in the data. When the data is validated and loaded the previous Jax pipeline can be retired.

###### ACCEPTANCE CRITERIA ########
- [x] metadata and molecular data validated
- [x] Jax loader retired

**Comment by zperova** [comment]
moving to the next sprint

**Comment by zperova** [comment]
related ticket: https://github.com/PDXFinder/pdxfinder/issues/190

**Comment by Afollet** [comment]
From the most recent communications between Steve and I:

I have updated the APIs.

1. Renamed endpoints.

2. Added new lines after column headers.

3. Deduplicated all data (I hope).

4. The missing patients are in the current patient endpoint (not sure what went wrong there I didn't change anything).

5. Checked a smattering of the models w/o samples they are models that don’t have sample data ie.

                http://tumor.informatics.jax.org/mtbwi/pdxDetails.do?modelID=J000106949

                Has nothing in the Molecular Data section.

 

TODO: Need to investigate the Patients w/o samples. I expect they correspond to models that have been discontinued, if so I will try to remove them from the patient API.

I will need to re-test these again and send feedback back to Steve.

**Comment by Afollet** [comment]
There is a brief bug to the code base I will need to write for the API to work. Steve is in development of the Omic API's

**Comment by zperova** [comment]
moving to the next sprint

**Comment by zperova** [comment]
@Afollet to forward email communication to @CsabaHalmagyi

**Comment by CsabaHalmagyi** [comment]
Asked Steve to clean up some of the tumor types in the sample sheet. The data loads fine, the indexer picked up 38 unmapped diagnosis combinations in the dataset.

**Comment by zperova** [comment]
WIll need to do 38 mappings once the branch is merged - @CsabaHalmagyi to let @zperova know when (to make a new ticket)
Moving to the next Sprint

**Comment by CsabaHalmagyi** [comment]
Removed custom JAX loader and supporting services

**Comment by CsabaHalmagyi** [comment]
Jax is now loaded with UPDOG, the custom loader was removed.

**Comment by zperova** [comment]
A call for a celebration! Thanks for all the work.
